.github/workflows/auto_assign.yml

```diff
@@ -0,0 +1,16 @@
+name: Auto Assign
+
+on:
+  pull_request:
+    types: [opened, ready_for_review]
+    branches: [main]
```

Comment:
Need to change branch from `main` to `master`, because you are using `master`

The same issue will be in ALL *.yaml files

Reply:
I have already renamed remote branch from master to main and it is working